tests/test_server.py

```diff
@@ -0,0 +1,105 @@
+import asyncio
+import unittest
+import json
+
+from unittest import mock
+
+
+class TestServer(unittest.TestCase):
+    MockedRequestHandler = None
+    srv = None
+    m = None
+    m_eof = None
```

Comment:
I think I don't have to. I think I just can assign values to it in setUp(), correct? I tried and it works. It will be great if you clarify this.